### PR TITLE
Add feature square

### PR DESCRIPTION
**Pull request by ivatavuk**

### CODE REVIEW

File: python/lust.py [hunk 1]
```diff
@@ -29,6 +29,24 @@ def __calculate_factorial(self, argument):
       result *= i
     return result
 
+class SquareCommand(LustObject):
```
Comment (AnaBatinovic) [inline python/lust.py:32]: Please document what are you doing here!

File: python/lust.py [hunk 2]
```diff
@@ -29,6 +29,24 @@ def __calculate_factorial(self, argument):
...
+    print(self.__calculate_square(argument))
+
+  def print_help(self):
+    print(" square <integer>")
+    print("   Calculates the square of <integer>.")
+
+  def __calculate_square(self, argument):
+    # Hmmm...
+   
+    return argument*argument
```
Comment (AnaBatinovic) [inline python/lust.py:48]: Use Python's (**) operator.

**Review by AnaBatinovic — APPROVED** [review]

File: python/lust.py (hunk 1 above)
Comment (ivatavuk) [reply python/lust.py:32]: necu!

File: python/lust.py (hunk 2 above)
Comment (ivatavuk) [reply python/lust.py:48]: necu!

**Pull request merged**